pilot/pkg/security/authn/utils/utils.go

```diff
@@ -85,10 +79,31 @@ func BuildInboundFilterChain(mTLSMode model.MutualTLSMode, node *model.Proxy,
 		}
 	}
 
+	allowedCiphers := []string{}
+	for _, c := range strings.Split(features.AllowedInboundCiphers, ",") {
```

Comment:
please do this 1 time at istiod initialization, rather than on every single cluster generated - which happens millions of times

Reply:
Revised to be at initialization.